### PR TITLE
Style dashboard

### PR DESCRIPTION
**Pull request by mrloureed**
- add tailwind
- add viam's prime core and theme
- create grid layout 6x4 
```
< {colspan-5 row-span-1} >< {row-span-4 >
< {no spans for camera feeds} >
```
- style existing components

Okay we're pretty close here 
image

https://www.figma.com/file/mtHyqhmsZNSPwfhIykixg5/Boat-Dashboard?type=design&node-id=836-2479&mode=design&t=BwZbY9sivLs49AC3-0

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,8 +3,9 @@
 	"version": "0.0.1",
 	"private": true,
 	"scripts": {
-		"dev": "vite dev",
-		"build": "vite build",
+		"dev": "npm-run-all --parallel dev:*",
```
Comment (mrloureed) [inline package.json:6]: Not sure if `npm-run-all` is a good solution here but it works

File: src/app.html [hunk 2]
```diff
@@ -7,6 +7,6 @@
 		%sveltekit.head%
 	</head>
 	<body data-sveltekit-preload-data="hover">
-		<div style="display: contents">%sveltekit.body%</div>
```
Comment (mrloureed) [inline src/app.html:10]: According to MDN this is not advised. Was there a reason for this @erh ?

Comment (erh) [reply src/app.html:10]: no clue, just think i copied this from somewhere

File: src/routes/+page.svelte [hunk 3]
```diff
@@ -794,88 +796,126 @@
...
-          <div class="data" >
-            <div>Depth ft</div>
-            {globalData.depth.toFixed(2)}
+        </div>
+      {/if}
+      {#if globalConfig.depthSensorName != ""}
+        <div class="flex gap-2 p-2 text-lg">
+          <div class="min-w-32">Depth</div>
+          <div>
+            <span class="font-bold">{globalData.speed.toFixed(2)}</span>
```
Comment (erh) [inline src/routes/+page.svelte]: this should be depth

**Review by erh — APPROVED** [review]

**Pull request merged**